website/frontend/src/pages/AboutUsPage.js

```diff
@@ -21,13 +21,13 @@ import HorizonteCiudadanoLogo from 'assets/img/partners/Horizonte Ciudadano Foun
 import KccaLogo from 'assets/img/partners/KCCA.png';
 import KiiraLogo from 'assets/img/partners/Kiira.png';
 import MappingForChangeLogo from 'assets/img/partners/Mapping for Change UK.png';
-import MinistriesLogo from 'assets/img/partners/Ministries.png'
+import MinistriesLogo from 'assets/img/partners/Ministries.png';
 import MUKLogo from 'assets/img/partners/MUK_Logo.png';
 import NEMALogo from 'assets/img/partners/NEMA.png';
 import SheffieldUniversityLogo from 'assets/img/partners/Sheffield University UK.png';
 import UNEPLogo from 'assets/img/partners/UNEP.png';
 import UniversityOfColoradoBoulderLogo from 'assets/img/partners/University of Colorado Boulder US.png';
-import UniversityOfFloridaLogo from 'assets/img/partners/University of Florida US.png' 
+import UniversityOfFloridaLogo from 'assets/img/partners/University of Florida US.png';
```

Comment:
Thanks @FaithDaka , I guess it is just safer not to have white spaces in file names. https://superuser.com/questions/29111/what-technical-reasons-exist-for-not-using-space-characters-in-file-names

Reply:
All the unused imports have been removed @Baalmart

Reply:
Thanks @FaithDaka